flows/vision-zero/cris_import/cris_import.py

```diff
@@ -512,7 +505,7 @@ def align_records(typed_token, dry_run):
     align_records_token = align_records(typed_token=typed_token, dry_run=dry_run)
 
     # push up the archives to s3 for archival
-    uploaded_archives_csvs = upload_csv_files_to_s3.map(extracted_archives)
+    # uploaded_archives_csvs = upload_csv_files_to_s3.map(extracted_archives)
```

Comment:
Since I'm out of the loop, is this a temporary change or are we stopping the S3 archives?

Reply:
Thanks Mike for catching that. I pushed up a change that reenables the task and restores its inter-task dependencies to get it back into the right part of the flow. 🙏🏻